src/strings.jl

```diff
@@ -25,7 +25,8 @@ const esriid2code = Dict(
   "World_Behrmann" => ESRI{54017},
   "World_Cylindrical_Equal_Area" => ESRI{54034},
   "World_Robinson" => ESRI{54030},
-  "World_Winkel_Tripel_NGS" => ESRI{54042}
+  "World_Winkel_Tripel_NGS" => ESRI{54042},
+  "ETRF2000-PL / CS92" => EPSG{2180}
```

Comment:
Please use the ESRI ID of the CRS, I believe they don't have whitespace. Also, please insert in alphabetical order.